### PR TITLE
feat: implement interaction between deno http sever and rust

### PR DESCRIPTION
**Pull request by hyeonLewis**
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix
- [x] Dependency, Environment, Build related update 
- [ ] Documentation
- [ ] Other tiny fix 
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Leave reference about this pull request, if exist (ex. discord message, google docs, etc)
- [x] If it is a core feature, I have added thorough tests.

## Checklist after creating a pull request
- [x] Mention reviews in discord channel directly

### CODE REVIEW

**Comment by chemonoworld** [comment]
포매팅 깨지는 듯요?

File: interact/src/lib.rs [hunk 1]
```diff
@@ -1 +1,209 @@
+extern crate dotenv;
+use anyhow::Result;
+use dotenv::dotenv;
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use std::{env, fmt::Debug};
 
+const HTTP_SERVER: &str = "http://localhost:8080/";
```
Comment (junha1) [inline interact/src/lib.rs:8]: 우리 HTTP 서버가 여러개 (프로덕션, 데브) 있을 가능성이 높아서 이거 상수로 박지말고 모든 함수에 그냥 argument로 넣는게 나을 것 같습니다.

File: interact/test_config_example.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
-    "account_private": "example.testnet.private",
-    "wasm_binary_path": "example/path"
-}
+    "test_local_node_url": "ws://127.0.0.1:9944",
+    "test_rococo_node_url": "wss://rococo-contracts-rpc.polkadot.io:443",
+    "test_shibuya_node_url": "wss://shibuya-rpc.dwellir.com:443",
+    "contract_address": "aeQtHkL7ENgfVYwiMZNYgz2GeRa75B2cSaVz1AFjZ9omRtv",
+    "account_public": "Wb3hWZDmozU9vUgM5b64q7AECoQgTLigFSKLz4jU9pYTonV",
+    "planck_to_one": 18
+}
```
Comment (junha1) [inline interact/test_config_example.json:8]: 뉴라인

**Review by junha1 — COMMENTED** [review]
구조상 TS에서 지원하는 컨트랙트만 제한적으로 되는거면 그냥 `contract_name` 대신에 가능한 컨트랙트를 다 enum으로 찍어주는게 좋을듯?

File: interact/test_config_example.json (hunk 2 above)
Comment (hyeonLewis) [reply interact/test_config_example.json]: fix

File: interact/src/lib.rs (hunk 1 above)
Comment (hyeonLewis) [reply interact/src/lib.rs]: fix

**Comment by chemonoworld** [comment]
check the above comment.
I think that using Enum instead of 'contract_name' is better, too

**Comment by chemonoworld** [comment]
LGTM

**Review by chemonoworld — DISMISSED** [review]

**Review by junha1 — DISMISSED** [review]

**Pull request merged**